IdentityServer/v7/docs/content/quickstarts/3_api_access.md

```diff
@@ -145,33 +145,35 @@ public class CallApiModel : PageModel
 }
 ```
 
-And update *src/WebClient/Pages/CallApi.cshtml* as follows:
+And update _src/WebClient/Pages/CallApi.cshtml_ as follows:
+
 ```html
-@page
-@model MyApp.Namespace.CallApiModel
+@page @model MyApp.Namespace.CallApiModel
 
 <pre>@Model.Json</pre>
 ```
 
-Also add a link to the new page in *src/WebClient/Shared/_Layout.cshtml* with the following:
+Also add a link to the new page in _src/WebClient/Shared/\_Layout.cshtml_ with the following:
+
 ```html
 <li class="nav-item">
   <a class="nav-link text-dark" asp-area="" asp-page="/CallApi">CallApi</a>
 </li>
 ```
 
-Make sure the *IdentityServer* and *Api* projects are running, start the
-*WebClient* and request */CallApi* after authentication.
+Make sure the _IdentityServer_ and _Api_ projects are running, start the
+_WebClient_ and request _/CallApi_ after authentication.
 
 ## Further Reading - Access token lifetime management
```

Comment:
this can probably go away / shortened in favour of the new QS

Reply:
Will do when the new qs is live.